src/codeu/chat/server/Server.java

```diff
@@ -112,6 +116,15 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
 
         final String name = Serializers.STRING.read(in);
         final User user = controller.newUser(name);
+        
+        String userAddCommand = "U-ADD ";
+        userAddCommand += user.id + " ";
+        userAddCommand += user.name + " ";
+        userAddCommand += user.creation;
+

```

Comment:
I think since a user has a creation time parameter, it's included in the google doc example so that a user can be created with it's unique id, name and creation time. Although, I do see where you're coming from and time might not need to be unique and can just be made as the current time that it is read from the file? Although, now that I'm thinking about it in order to replicate the userByTime store, maybe it should have the original time?